docs/ref/models/instances.txt

```diff
@@ -309,6 +309,21 @@ pass a dictionary mapping field names to errors::
         'pub_date': ValidationError(_('Invalid date.'), code='invalid'),
     })
 
+.. note::
+   If the model is used in a model form anywhere on your site, you
+   must only use field names in your dictionary that will exist on all
+   forms, otherwise Django will raise a ``ValueError`` exception because
+   it will be unable to tie the errors to their fields.
+
+   The ``clean_fields()`` method receives the list of fields that should
+   be excluded from validation. Look into overriding this method and
```

Comment:
When you say "Look into overriding", is this meant as a way to workaround the issue described in the previous paragraph? If so, I would make the wording more explicit about that.

A code example could be quite useful to help illustrate the ideas you've described.